code/workspaces/web-app/src/containers/dataStorage/DataStorageContainer.js

```diff
@@ -102,8 +110,11 @@ class DataStorageContainer extends Component {
 
   componentWillMount() {
```

Comment:
Seeing this reminds me that this function is now throwing warnings as it is deemed unsafe. Should we be looking to a different lifecycle hook as part of this widescale refactor?

Reply:
Perfect, I will add a ticket to the backlog for it.